test/e2e/apimachinery/webhook.go

```diff
@@ -793,7 +793,6 @@ func deployWebhookAndService(f *framework.Framework, image string, certCtx *cert
 				"webhook",
 				"--tls-cert-file=/webhook.local.config/certificates/tls.crt",
 				"--tls-private-key-file=/webhook.local.config/certificates/tls.key",
-				"--alsologtostderr",
```

Comment:
I'm dropping this because I got the following error from container logs after updating from agnhost `2.40` -> `2.43`

> kubectl logs sample-webhook-deployment-57d7d95b65-xv4t5 -n webhook-3757
Error: unknown flag: --alsologtostderr